odoo/tools/convert.py

```diff
@@ -31,6 +31,11 @@
 from .safe_eval import safe_eval as s_eval
 safe_eval = lambda expr, ctx={}: s_eval(expr, ctx, nocopy=True)
 
+
+# Match any QUOTED first cell of a row that starts with #, regardless of newlines
+INVALID_CSV_CELL_RE = re.compile(b'^"#', re.MULTILINE)
```

Comment:
I really don't like this. If we have to introduce limitation on the CSV contents it's better not to have the feature at all.

Reply:
@odony well, it is difficult to hack something around the built-in `csv.reader` without introducing some limitations.  We have not seen a way to implement the spec without rewriting a CSV reader.  Do you think it is worth the effort?